### ISSUE TITLE
The recommendation database for Klee is incorrect

### ISSUE THREAD

**Issue by breezewish** [ideless/artifact#32]
According to https://bbs.nga.cn/read.php?pid=635740877 Klee does not need EM and ER.

Seems that the characterData.json is generated from something else? It would be great if you can show me how it is generated so that I can contribute as well.

**Comment by ideless** [comment]
no... `weight` is based on https://bbs.nga.cn/read.php?pid=635740877 but manually modified by myself in a somewhat casual manner. If you need "the criteria", 1 stands for important, 0.5 for good but not essential, 0.25 for better than nothing, and 0 for useless. For Klee, I think EM and ER are better than nothing. You can customize these builds on the website, but I agree that the default build should be better. I'm open to better ideas.

**Comment by breezewish** [comment]
> no... `weight` is based on https://bbs.nga.cn/read.php?pid=635740877 but manually modified by myself in a somewhat casual manner. If you need "the criteria", 1 stands for important, 0.5 for good but not essential, 0.25 for better than nothing, and 0 for useless. For Klee, I think EM and ER are better than nothing. You can customize these builds on the website, but I agree that the default build should be better. I'm open to better ideas.

感谢回复，听起来是合理的。

另外还有一个建议，看起来很多模型的 CR% 和 CD% 用的是 1.0，但 ATK% 用的是 0.5，这是否意味着最终计算期望词条时，2 个 ATK% 词条才能等效于 1 个 CR%/CD% 词条呢？这个比例会不会太低了？例如占卜铺里挺多时候 ATK% 可以达到 CR%/CD% 的 75% 以上：

<img width="701" alt="image" src="https://user-images.githubusercontent.com/1916485/236598552-86075186-fe53-443e-8e0a-1ca36b904ca8.png">

**Comment by ideless** [comment]
我猜测这张图应该反应的是效用（伤害）函数在某一点（当前圣遗物面板）处的偏导，但效用函数通常是一个非线性函数，对于另外一个圣遗物面板攻击力加成可能就没那么重要了。可能取所有点处的平均值会合理一些。
